CalDAV.NET/Client.cs

```diff
@@ -150,10 +150,10 @@ private async Task<ICalendar> GetCalendarWithUriAsync(string uri)
 
             // check if resource really is a calendar
             var contentType = resource?.Properties.FirstOrDefault(x => x.Key.LocalName == "getcontenttype");
-            if (contentType.HasValue == false || contentType.Value.Value != "text/calendar")
-            {
-                return null;
-            }
+            // if (contentType.HasValue == false || contentType.Value.Value != "text/calendar")
```

Comment:
Hello, thank you for your pull request.

Can you explain to me why you did remove the content value check? And is there a reason why you did comment it instead of removing the lines?